modules/openxr/editor/openxr_select_action_dialog.cpp

```diff
@@ -0,0 +1,135 @@
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#include "openxr_select_action_dialog.h"
+#include "editor/editor_node.h"
+
+void OpenXRSelectActionDialog::_bind_methods() {
+	ADD_SIGNAL(MethodInfo("action_selected", PropertyInfo(Variant::STRING, "action")));
+}
+
+void OpenXRSelectActionDialog::_notification(int p_what) {
+	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE:
+		case NOTIFICATION_THEME_CHANGED: {
+			scroll->add_theme_style_override("bg", get_theme_stylebox(SNAME("bg"), SNAME("Tree")));
+		} break;
+	}
+}
+
+void OpenXRSelectActionDialog::_on_select_action(const String p_action) {
+	if (selected_action != "") {
+		NodePath button_path = action_buttons[selected_action];
+		Button *button = static_cast<Button *>(get_node(button_path));
+		if (button != nullptr) {
+			button->set_flat(true);
+		}
+	}
+
+	selected_action = p_action;
+
+	if (selected_action != "") {
+		NodePath button_path = action_buttons[selected_action];
+		Button *button = static_cast<Button *>(get_node(button_path));
+		if (button != nullptr) {
+			button->set_flat(false);
+		}
+	}
+}
+
+void OpenXRSelectActionDialog::open() {
+	ERR_FAIL_COND(action_map.is_null());
+
+	// out with the old...
+	while (main_vb->get_child_count() > 0) {
+		memdelete(main_vb->get_child(0));
+	}
+
+	selected_action = "";
+	action_buttons.clear();
+
+	Array action_sets = action_map->get_action_sets();
+	for (int i = 0; i < action_sets.size(); i++) {
+		Ref<OpenXRActionSet> action_set = action_sets[i];
+
+		Label *action_set_label = memnew(Label);
+		action_set_label->set_text(action_set->get_localized_name());
+		main_vb->add_child(action_set_label);
+
+		Array actions = action_set->get_actions();
+		for (int j = 0; j < actions.size(); j++) {
+			Ref<OpenXRAction> action = actions[j];
+
+			HBoxContainer *action_hb = memnew(HBoxContainer);
+			main_vb->add_child(action_hb);
+
+			Control *indent_node = memnew(Control);
+			indent_node->set_custom_minimum_size(Size2(10.0, 0.0));
+			action_hb->add_child(indent_node);
+
+			Button *action_button = memnew(Button);
+			String action_name = action->get_name_with_set();
+			Vector<Variant> binds;
+			binds.push_back(action_name);
+			action_button->set_flat(true);
+			action_button->set_text(action->get_name() + ": " + action->get_localized_name());
+			action_button->connect("pressed", callable_mp(this, &OpenXRSelectActionDialog::_on_select_action), binds);
+			action_hb->add_child(action_button);
+
+			action_buttons[action_name] = action_button->get_path();
+		}
+	}
+
+	popup_centered();
+}
+
+void OpenXRSelectActionDialog::ok_pressed() {
+	if (selected_action == "") {
+		return;
+	}
+
+	emit_signal("action_selected", selected_action);
+
+	hide();
+}
+
+OpenXRSelectActionDialog::OpenXRSelectActionDialog(Ref<OpenXRActionMap> p_action_map) {
+	action_map = p_action_map;
+
+	set_title("Select an action");
```

Comment:
```suggestion
	set_title(TTR("Select an action"));
```
Use `TTR` to make editor strings localizable.